includes/definitions.inc.php

```diff
@@ -888,8 +888,8 @@
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
 $os = 'fabos';
-$config['os'][$os]['text']             = 'Brocade FabricOS';
-$config['os'][$os]['type']             = 'network';
+$config['os'][$os]['text']             = 'FabricOS';
+$config['os'][$os]['type']             = 'storage';
```

Comment:
because they are FC switches and usually they get under the administration of storage "department" not networking